herddb-core/src/main/java/herddb/file/FileDataStorageManager.java

```diff
@@ -944,6 +944,25 @@ public void writeIndexPage(
             }
 
         } catch (IOException err) {
+            LOGGER.log(Level.SEVERE, "Failed to write on path: {0}", pageFile);
```

Comment:
I am not sure we want to keep this debug.

Maybe you can create an helper method and enable it with a system property.
I am -0 on keeping it

Reply:
Actually it was a log added for debugging... then at commit time I thought to left it there (lines after line 947).

I'm not sure if let it there. It shouldn't happen but knowing that the parent path didn't exists helped to found the issue.